core/cmd/cosmos_transaction_commands_test.go

```diff
@@ -0,0 +1,145 @@
+//go:build integration
+
+package cmd_test
+
+import (
+	"flag"
+	"strconv"
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli"
+
+	cosmosclient "github.com/smartcontractkit/chainlink-cosmos/pkg/cosmos/client"
+	cosmosdb "github.com/smartcontractkit/chainlink-cosmos/pkg/cosmos/db"
+
+	"github.com/smartcontractkit/chainlink/core/chains/cosmos/cosmostxm"
+	"github.com/smartcontractkit/chainlink/core/chains/cosmos/denom"
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/cosmostest"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/cosmoskey"
+)
+
+func TestClient_SendCosmosCoins(t *testing.T) {
+	t.Skip("requires wasmd")
```

Comment:
Don't like to see skipped tests getting merged

Reply:
Maybe a build tag instead? `//go:build wasmd`